pegasus/sites.v3/code.org/views/theme_common_head_after.haml

```diff
@@ -1,4 +1,7 @@
-%script=view 'google_analytics.js'
+- cookie_key = '_user_type' + (rack_env == :production ? '' : "_#{rack_env.to_s}")
```

Comment:
Why is the `to_s` necessary? Doesn't the "#{...}" already cause string interpolation?